### ISSUE TITLE
Previous page navigation broken

### ISSUE THREAD

**Issue by wolfgangmm** [eXistSolutions/hsg-shell#74]
This used to work until recently, but currently, navigating back skips one section. For example, if you start at

http://localhost:8080/exist/apps/hsg-shell/historicaldocuments/frus1952-54v02p1/d6

and press the back arrow, you are taken to d4 instead of d5.

**Comment by joewiz** [comment]
I'm not able to reproduce your result.  I get the expected result, d5.

**Comment by joewiz** [comment]
Actually, I am seeing the behavior.  I don't see it if I go directly to d6 via the URL (thus using the full HTML load).  But if I arrive at a page via the prev/next arrow buttons (using the frus-ajax.xql query), that page has the wrong back link.  I'm not seeing this behavior affecting the forward navigation.  Thus I think the issue must be in the way frus-ajax.xql is looking up the previous div.  I did modify some of this behavior recently, so it's probably something I introduced or a side effect of the change I made.  I should be able to commit a fix shortly.
